[DOCS] Clarify alert de-duplication logic around configured 'additional look-back time' (#1125)
elastic/security-docs: docs/detections/rules-ui-monitor.asciidoc

for a small number of rules, you can increase those rules'
Additional look-back time (*Detect* -> *Rules* -> the rule's *All actions* button (*...*) -> *Edit rule settings* -> *Schedule* -> *Additional look-back time*).

> It is recommended to set the `Additional look-back time` to at

Reviewer: ```suggestion
It's recommended to set the `Additional look-back time` to at
```

> least 1 minute. This ensures there are no missing alerts when a rule does not

Reviewer: ```suggestion
least 1 minute. This ensures there are no missing alerts when a rule doesn't
```
